safe-global/safe-wallet-web
# feat: add LI.FI widget

## What it solves

We want to natively facilitate bridging in the UI. This adds the LI.FI widget, accessed via a new "Bridge" sidebar element.

## How this PR fixes it

A new sidebar element ("Bridge") opens a new `/bridge` route, displaying the LI.FI widget.

## How to test it

Open the "Bridge" section of a Safe and observe the widget, with the ability to bridge.

## Screenshots

![image](https://github.com/user-attachments/assets/adaebb02-2a1a-4874-ac31-dbe432951fc5)

![image](https://github.com/user-attachments/assets/d8ba218f-a90a-4207-bf79-f5943ce402cd)

## Checklist
